index.bs

```diff
@@ -3657,7 +3688,89 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
+
+The value of the `BS` [=flag=] may change over time based on the current state of the [=Public Key Credential Source|credential source=]. <a href="#table-backupStates">Table <span class="table-ref-following"/></a> below defines
+valid combinations and their meaning.
+
+<figure id="table-backupStates" class="table">
+    <table class="complex data longlastcol">
+            <tr>
+            <th>`BE`</th>
+            <th>`BS`</th>
+            <th>Description</th>
+        </tr>
+        <tr>
+            <td>`0`</td>
+            <td>`0`</td>
+            <td>
+                The credential is a [=single-device credential=].
+            </td>
+        </tr>
+        <tr>
+            <td>`0`</td>
+            <td>`1`</td>
+            <td>
+                This combination is not allowed.
+            </td>
+        </tr>
+        <tr>
+            <td>`1`</td>
+            <td>`0`</td>
+            <td>
+                The credential is a [=multi-device credential=] and is not yet backed up.
+            </td>
+        </tr>
+        <tr>
+            <td>`1`</td>
+            <td>`1`</td>
+            <td>
+                The credential is a [=multi-device credential=] and is backed up.
+            </td>
+        </tr>
+    </table>
+    <figcaption>
+        `BE` and `BS` [=flag=] combinations
+    </figcaption>
+</figure>
+
+It is RECOMMENDED that [=[RPS]=] store the most recent value of these [=flags=] with the [=user account=] for future evaluation.
+
+The following is a non-normative, non-exhaustive list of how [=[RPS]=] might utilize these [=flags=]:
+
+    - Requiring additional [=authenticators=]:
 
+        When `BE` [=flag=] is set to `0`, the [=generating authenticator=] will never allow the credential to transition from a 
+        [=single-device credential=] to a [=multi-device credential=]. 
+        
+        A [=single-device credential=] is not resilient to single device loss. [=[RPS]=] SHOULD ensure that a [=user account=] 
+        has additional [=authenticators=] [=registration ceremony|registered=] and/or an account recovery process in place.
+
+        For example, the user could be prompted to set up an additional [=authenticator=], such as a [=roaming authenticator=] or an 
+        [=authenticator=] that is capable of [=multi-device credentials=].
+    
+    - Upgrading a user to a password-free account:
+
+        When the `BS` [=flag=] changes from `0` to `1`, the [=authenticator=] is signaling that the [=credential=] is backed up and is protected from single device loss.
+        
+        A [=Relying Party=] may decide to prompt the user to upgrade their account security and remove their password.
+    
+    - Adding an additional factor after a state change:
+
+        When the `BS` [=flag=] changes from `1` to `0`, the [=authenticator=] is signaling that the [=credential=] is no longer backed up, 
```

Comment:
(non blocking) In my mind, I thought this transition was going to be banned for simplicity, but I haven't been following all the discussions on this.